codersdk/audit.go

```diff
@@ -114,6 +115,8 @@ func (a AuditAction) Friendly() string {
 		return "logged out"
 	case AuditActionRegister:
 		return "registered"
+	case AuditActionRequestOneTimePasscode:
+		return "one time passcode requested"
```

Comment:
nit: what do you think about "password reset"?

Reply:
Happy to make this change

Reply:
Have made these changes!